tools/accuracy_checker/sample/sample_config.yml

```diff
@@ -4,7 +4,7 @@ models:
     # list of launchers for your topology.
     launchers:
       # launcher framework (e.g. caffe, openvino)
-      - framework: openvino
+      - framework: dlsdk
```

Comment:
Please return back, as I said in the issue, it is not a bug!!! We intentionally change framework for upcoming release.

Reply:
fixed

Reply:
as I can see, not, it is not fixed

Reply:
please revert changes regarding ac sample config, it is very strange when even you separe request on 2 parts you still have AC modifications in request with demo

And you also mark that you fixed this issue but do nothing.... your changes still in PR